src/main/resources/META-INF/plugin.xml

```diff
@@ -1,41 +1,16 @@
+<!-- Plugin Configuration File. Read more: https://plugins.jetbrains.com/docs/intellij/plugin-configuration-file.html -->
 <idea-plugin>
   <id>io.pivotal.jasmine</id>
   <name>Jasmine</name>
-  <version>1.2</version>
   <vendor email="jasmine-js@googlegroups.com" url="http://jasmine.github.io">Jasmine</vendor>
 
-  <depends>com.intellij.modules.lang</depends>
-  <depends>com.intellij.modules.json</depends>
```

Comment:
On my machine, this breaks installation in WebStorm 2025.1: "Plugin 'IdeaJasmine' is compatible with IntelliJ IDEA only because it doesn't define any explicit module dependencies".

If I put those two <depends> lines back, it installs and runs.